dask/dataframe/core.py

```diff
@@ -696,6 +696,99 @@ def to_csv(self, filename, **kwargs):
         from .io import to_csv
         return to_csv(self, filename, **kwargs)
 
+    @wraps(pd.DataFrame._get_numeric_data)
+    def _get_numeric_data(self, how='any', subset=None):
+        return map_partitions(pd.DataFrame._get_numeric_data, self.columns, self)
+
+    def _validate_axis(self, axis=0):
+        if axis not in (0, 1, 'index', 'columns', None):
+            raise ValueError('No axis named {0}'.format(axis))
+        # convert to numeric axis
+        return {None: 0, 'index': 0, 'columns': 1}.get(axis, axis)
+
+    @wraps(pd.DataFrame.sum)
+    def sum(self, axis=None):
+        axis = self._validate_axis(axis)
+        if axis == 1:
+            f = lambda x: x.sum(axis=1)
+            return map_partitions(f, self.columns, self)
+        else:
+            chunk = lambda df: df.sum()
+            agg = lambda df: df.groupby(level=0).sum()
+            return aca([self], chunk=chunk, aggregate=agg, columns=self.columns)
```

Comment:
Same here actually, it seems like we want the result to be a `Series` with `name=None`